techniques/failures/F69.html

```diff
@@ -149,22 +146,22 @@ <h3>Example 2: Truncated text</h3>
 
 
 
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
          <div class="note">
-				<p class="note-title marker">Note</p>
+				<p class="note-title marker">注記</p>
 				<div>
-            <p>The Working Group has discovered many misunderstandings about how to test this failure. We are planning to revise this failure in a future update. Until then, if the content passes the success criterion using any of the listed sufficient techniques, then it does not meet this failure.</p>
+            <p>AG ワーキンググループは、この失敗例の検証方法に多くの誤解があることを確認している。この失敗例の文書を次回の更新時に修正する予定である。それまでは、コンテンツが「十分な達成方法」のいずれかを用いて達成基準を満たしていれば、この失敗例には該当しないと考えてよい。</p>
          </div>
 			</div>
          <ol>
-            <li>Increase the text size of the content by 200%.</li>
-            <li>Check that no text is clipped, truncated, or obscured.</li>
+            <li>コンテンツのテキストの文字サイズを 200 ％拡大する。</li>
+            <li>テキストが切り取られたり、端が欠けたり、又は覆い隠されたりしていない。</li>
```

Comment:
check that ... を訳出しました。
```suggestion
            <li>テキストが切り取られたり、端が欠けたり、又は覆い隠されたりしていないことをチェックする。</li>
```